syntax/pandoc.vim

```diff
@@ -81,7 +81,7 @@ endif
 if !exists("g:pandoc#syntax#conceal#backslash") 
     let g:pandoc#syntax#conceal#backslash = 0
 endif
-"}}}2
+"}}}2"
```

Comment:
This is an end delimiter for folding, not a string, and does not need this ending quote.